pkg/reconciler/dependencybuild/scripts/gradle-build.sh

```diff
@@ -10,8 +10,6 @@ cp -r /maven-artifacts/.m2/* "$HOME/.m2/" || true
 cat > "${GRADLE_USER_HOME}"/gradle.properties << EOF
 org.gradle.caching=false
 org.gradle.console=plain
-# This prevents the daemon from running (which is unnecessary in one-off builds) and increases the memory allocation
-org.gradle.daemon=false
```

Comment:
Gradle recommendation is to keep the daemon running now ( e.g. https://docs.gradle.org/current/userguide/gradle_daemon.html#continuous_integration and this is also what Travis CI does).